pkg/tests/common/appstudio.go

```diff
@@ -19,26 +23,35 @@ var (
 var _ = framework.CommonSuiteDescribe("Red Hat App Studio common E2E", func() {
 	defer g.GinkgoRecover()
 	commonController, err := commonCtrl.NewSuiteController()
-	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	Expect(err).NotTo(HaveOccurred())
 
 	g.Context("Argo CD", func() {
 		for _, component := range AppStudioComponents {
 			g.It(component+" status", func() {
 				componentStatus, err := commonController.GetAppStudioComponentStatus(component, AppStudioComponentsNamespace)
-				gomega.Expect(err).NotTo(gomega.HaveOccurred())
-				gomega.Expect(componentStatus.Health.Status).To(gomega.Equal(health.HealthStatusHealthy))
+				Expect(err).NotTo(HaveOccurred())
+				Expect(componentStatus.Health.Status).To(Equal(health.HealthStatusHealthy))
 
 			})
 		}
 	})
+	defer g.GinkgoRecover()
 
 	g.Context("ClusterTasks:", func() {
-		for _, pipelineName := range AppStudioPipelinesNames {
-			g.It("Check if "+pipelineName+" clustertask is pre-created", func() {
-				p, err := commonController.GetClusterTask(pipelineName, PipelinesNamespace)
-				gomega.Expect(err).NotTo(gomega.HaveOccurred())
-				gomega.Expect(p.Name).To(gomega.Equal(pipelineName))
+		g.It("Check if AppStudio ClusterTasks are precreated", func() {
+			err := wait.PollImmediate(30*time.Second, 10*time.Minute, func() (done bool, err error) {
```

Comment:
the first parameter is the interval of polling - 30seconds is a quite high interval. I would expect something like 100 milliseconds - or something like that.
Also, the timeout is pretty high - I'm not sure if anyone would want to wait 10 minutes - imagine the situation when there is something wrong and the ClusterTasks are not created. Waiting 10 minutes extremely prolongs the feedback loop.

Reply:
I'm thinking about 3 minutes and 100Miliseconds interval... WDYT @MatousJobanek ?

Reply:
Yeah, 3 minutes sounds much better. But it would be nice to have some data for this number. If the ClusterTasks are created in almost all cases under a few seconds, then you could set the timeout to 1 minute.